go/vt/vttablet/tabletserver/tx_engine.go

```diff
@@ -94,6 +96,12 @@ type TxEngine struct {
 	dxNotify     func()
 }
 
+const (
+	TwoPCAllowed_SemiSync = iota
+	TwoPCAllowed_TabletControls
+	TwoPCAllowed_Len
```

Comment:
I'm still not 100% sure what these mean as I go through the code (hopefully I do by the end). I think it's worth a comment.

Reply:
Done